src/routes/_actions/timeline.js

```diff
@@ -49,7 +104,7 @@ async function fetchTimelineItems (instanceName, accessToken, timelineName, last
     try {
       console.log('fetchTimelineItemsFromNetwork')
       items = await fetchTimelineItemsFromNetwork(instanceName, accessToken, timelineName, lastTimelineItemId)
-      /* no await */ storeFreshTimelineItemsInDatabase(instanceName, timelineName, items)
+      await storeFreshTimelineItemsInDatabase(instanceName, timelineName, items)
```

Comment:
I was trying to be clever by avoiding this `await` and relying on the sync db cache to be updated first, but this is really fragile and it's not a big deal to wait on IndexedDB after we finished waiting on the network